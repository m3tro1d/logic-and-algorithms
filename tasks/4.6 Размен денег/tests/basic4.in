1500 20
1000 943 904 870 822 754 689 659 606 571 521 464 423 373 329 250 241 170 114 70
