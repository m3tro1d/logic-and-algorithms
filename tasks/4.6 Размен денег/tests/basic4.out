2 3
1000 1
250 2
